houston/types.go

```diff
@@ -312,6 +314,16 @@ type FeatureFlags struct {
 	NamespaceFreeFormEntry bool `json:"namespaceFreeFormEntry"`
 }
 
+// Team contains all components of an Astronomer Team
+type Team struct {
+	ID           string        `json:"id"`
+	Name         string        `json:"name"`
+	SortID       int           `json:"sortid"`
```

Comment:
is it called `sortid` in the backend or is it a typo for `sortId` ?

Reply:
ill double check, kinda got this from other pr, thanks!

Reply:
it was a typo good catch!